crates/typst/src/layout/place.rs

```diff
@@ -1,11 +1,23 @@
 use crate::foundations::{elem, scope, Cast, Content, Smart};
 use crate::layout::{Alignment, Em, Length, Rel};
 
-/// Places content at an absolute position.
+/// Places content relatively to its parent container.
```

Comment:
It'd be nice to mention the "absolute position" terminology somewhere below. I agree it's not fully correct, but it's what people are used to read, so it's usually easier to associate with behavior from elsewhere. Maybe a short paragraph or sentence somewhere mentioning that "in the absence of a parent container, the content will be placed absolutely, that is, relative to the current page" could help.

Reply:
I'd also consider adding a link like `[container]($block)`. Though it may be a bit misleading as blocks are not the only containers (though, in practice, everything becomes a block at some point...). But would be a bit better than the status quo, I think.

Could also merge this with my page suggestion and mention "relatively to the parent container, such as an enclosing `[block]($block)`, or, in the absence of one, absolutely, that is, relatively to the current page.", or something similar.

Reply:
`[block]($block)` can be shortened to `[block]` btw (and `` [`block`] `` works as well)

Reply:
When first trying to use `place` I was super confused by this mention of absolute positioning, because the placement is relative to the container, then relative to the alignment in the container, and indeed the function takes offsets `dx` and `dy` rather than absolute positions `x` and `y`, and even in the outer scope you are relative to the text area (i.e. it respects margins) rather than the page itself.

But it's true that `place` is also the way to implement absolute positioning :-) So I added a paragraph about that, together with links to typical containers.